test/unit/vl-loader.test.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-loader.js"></script>
+</head>
+
+<body>
+  <test-fixture id="vl-loader-fixture">
+    <template>
+      <vl-loader></vl-loader>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="vl-loader-with-slot-fixture">
+    <template>
+      <vl-loader>
+        <span>Dit is een slot tekst</span>
+      </vl-loader>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<vl-loader>', () => {
+      const should = chai.should();
+
+      test('bevat gecentreerd een loader indicator en een standaard tekst', () => {
+        const element = fixture('vl-loader-fixture');
+        const container = element.shadowRoot.querySelector('div');
```

Comment:
niet beter de div ook een class container of zo geven dan. zoek mij "een" div vind ik altijd zowat vervelend.

Reply:
De div is onderdeel van de shadow DOM die later in de test gecontroleerd op de aanwezigheid van de class. Een alternatief kan zijn om een should.exist te gebruiken en meteen a.d.h.v. van de center class de container te selecteren. Ik heb dit voorzien.